spring/spring4/spring4-remote/src/test/java/org/infinispan/spring/provider/sample/SampleHotrodServerLifecycleBean.java

```diff
@@ -46,8 +47,8 @@ public void afterPropertiesSet() throws Exception {
       cacheManager.defineConfiguration(remoteBackupCacheName, HotRodTestingUtil.hotRodCacheConfiguration().build());
       cacheManager.defineConfiguration(customCacheName, HotRodTestingUtil.hotRodCacheConfiguration().build());
       HotRodServerConfigurationBuilder hcb = new HotRodServerConfigurationBuilder();
-      hcb.port(15233);
-      hotrodServer = HotRodClientTestingUtil.startHotRodServer(cacheManager, hcb);
+      hcb.port(PORT);
+      hotrodServer = HotRodClientTestingUtil.startHotRodServer(cacheManager, PORT, hcb);
```

Comment:
@danberindei +1. But I'd prefer to do it in a separate JIRA (so I created [ISPN-8017](https://issues.jboss.org/browse/ISPN-8017))